src/components/content/document/patient-document-filters.tsx

```diff
@@ -3,7 +3,17 @@ import { isEqual, uniqWith } from "@/utils/nodash";
 
 export const applyDocumentFilters = (data: fhir4.DocumentReference[]) => {
   const documentModel = data.map((document) => new DocumentModel(document));
-  const documentData = uniqWith(documentModel, (a, b) =>
+
+  const actualDocument = documentModel.filter(
+    (document) =>
+      document.category &&
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+      document.category.filter(
+        (category) => category.coding && category.coding.length > 1
+      )
```

Comment:
```suggestion
 document.category.filter(
        (category) => category.coding && category.coding.length > 1
      ).length > 0
```

I believe you want something like this. The reason eslint is warning you about this is because since you already checked if document.category is truthy and in the next line you re doing the same thing. You probably want to check that it has actual elements inside of it